commons/src/main/kotlin/com/simplemobiletools/commons/compose/theme/Dimensions.kt

```diff
@@ -0,0 +1,33 @@
+package com.simplemobiletools.commons.compose.theme
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.ProvidableCompositionLocal
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.ui.unit.dp
+import com.simplemobiletools.commons.compose.theme.model.Dimensions
+
+val Dimens: Dimensions
```

Comment:
Don't create this type of shortcuts, it's a double edged sword, it's better to use `val dimens = LocalDimensions.current` IMO

Reply:
To be honest, I would have liked something like `MaterialTheme` object, that encapsulates a couple of properties that act like this, making it much harder to accidentally import these shortcuts accidentally, but I was not sure how to name it so I left it like this for now. I can change to `dimens`, unless you have a suggestion for an object name that would hold this and maybe some other similar properties in the future.